Documentation/operations/upgrade.rst

```diff
@@ -377,7 +377,10 @@ Annotations:
        egressDeny:
        - toCIDR:
          - 0.0.0.0/0
-             
+
+* IPv6 on ``cilium_host`` now will be assigned from IPAM pool, rather than using
+  the same IPv6 as native host interface, such as ``eth0``. This means
+  ``cilium_host``'s IPv6 changes after upgrade, as well as related routing rules.
```

Comment:
Maybe add links to related GH issues (notably https://github.com/cilium/cilium/issues/14509)